catalog/schematron/catalog-schematron-plugin/src/main/java/ddf/services/schematron/SchematronValidationService.java

```diff
@@ -266,6 +273,15 @@ private SchematronReport generateReport(String metadata, Templates validator)
         return report;
     }
 
+    /**
+     * Replace tabs, literal carriage returns, and newlines with a single whitespace
+     * @param input
+     * @return
+     */
+    static String sanitize(final String input) {
```

Comment:
If somebody changed `sanitize` accidentally, I'm pretty sure it would break the tests as well.